README.md

```diff
@@ -114,6 +114,8 @@ Besides `base_controller_class`, you can also set the following for `MissionCont
 - `scheduled_job_delay_threshold`: the time duration before a scheduled job is considered delayed. Defaults to `1.minute` (a job is considered delayed if it hasn't transitioned from the `scheduled` status 1 minute after the scheduled time).
 - `show_console_help`: whether to show the console help. If you don't want the console help message, set this to `false`—defaults to `true`.
 - `backtrace_cleaner`: a backtrace cleaner used for optionally filtering backtraces on the Failed Jobs detail page. Defaults to `Rails::BacktraceCleaner.new`. See the [Advanced configuration](#advanced-configuration) section for how to configure/override this setting on a per application/server basis.
+- `filter_arguments`: an array of job argument keys that you want to filter out in the UI. This is useful for hiding sensitive user data. You can also override this option for each application. Currently, only root-level hash keys are supported, and it only works with the `Resque` and `SolidQueue` adapters. See the [Advanced configuration](#advanced-configuration) section for an example.
```

Comment:
I think mentioning that it only works with the `Resque` and `SolidQueue` adapters is not necessary because only those two adapters are supported. There's also the `async` adapter, but that's just to prevent it from breaking in development environment where that's the default adapter. No jobs can be inspected there 😅